readme.md

```diff
@@ -168,72 +384,67 @@ var Hello = React.createClass({
+* Define a Post constructor in `index.js` that has the below properties. Then use it to generate an instance of a Post with the content of your choosing.
+  1. `title`
+  2. `author`
+  3. `body`
+  4. `comments` (array of strings)
 * Render these properties using a Post component.
-* The HTML/CSS composition of your Post is up to you.
-
-### Solution
+* The HTML (or more accurately, JSX) composition of your Post is up to you.
 
-[https://gist.github.com/amaseda/efeb9d6ee266f8fa3178](https://gist.github.com/amaseda/efeb9d6ee266f8fa3178)
+#### [Solution](https://gist.github.com/amaseda/efeb9d6ee266f8fa3178)
 
-## Nested Components (5 / 50)
+## Nested Components
 
-What problems did you encounter when trying to add multiple comments to your Post?
+**Q:** What problems did you encounter when trying to add multiple comments to your Post?
 * It would be a pain to have to explicitly define every comment inside of `<PostView />`, especially if each comment itself had multiple properties.
-* The solution? **Nested components**.
 
-We can nest Comment components within a PostView component
+We can nest Comment components within a PostView component.
 * We create these comments the same way we did with posts: `.createClass` and `.render`
 * Then we can reference a comment using `<Comment />` inside of PostView's render method.
 
-## Exercise: Add Nested Comments To Blog (10 / 60)
+## Exercise: Add Nested Comments To Blog
 
-1. Create a `CommentView` component in the same way we did for `PostView`.
-* Your `CommentView` render method should render a `commentBody` property.
+1. Create a `CommentView` component in the same way we did for `PostView`. Its `render` method should render a `commentBody` property.
 
 2. Amend your `PostView`'s render method so that its return value generates three `<CommentView />` elements.
-* Make sure to pass in the comment body as an argument to each component.
 
-### Solution
-
-[https://gist.github.com/amaseda/0eba4f73b1ec8073ea0a](https://gist.github.com/amaseda/0eba4f73b1ec8073ea0a)
+> Make sure to pass in the comment body as an argument to each component.
 
-## Break (10 / 70)
+#### [Solution](https://gist.github.com/amaseda/0eba4f73b1ec8073ea0a)
 
-## State (15 / 85)
+## State
 
-We already went over properties.
-* The thing about properties is, you can't change them.
+So we know about React properties. The thing about properties is, you can't change them!
 * What do we do about values that need to be changed after a component is rendered?
 * That's where **state** comes in.
 
 Values stored in a component's state are mutable attributes.
-* Like properties, we can access state values using `this.state.val`
-* Setting up and modifying state is not as straightforward as properties. Requires multiple methods
+* Like properties, we can access state values using `this.state.val`.
+* Setting up and modifying state is not as straightforward as properties. It requires multiple methods that we must define ourselves.
```

Comment:
It involves explicitly declaring the mutation, and then defining methods to define how to update our state....